operations/helm/charts/mimir-distributed/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
-version: 5.5.0-weekly.309
-appVersion: r309
+version: 5.5.0-rc.0
+appVersion: 2.14-rc.0
```

Comment:
Shouldn't the app version is 2.14.0-rc.0?

Reply:
Ooops, you're right. Thank you